tools/jdk/remote_java_tools_aliases.bzl

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A collection of macros that wrap native Bazel rules and select the given target
+# from an external java tools repository based on the platform.
+
+def _get_args(target, attr, **kwargs):
+    workspace_target_dict = {
+        "//src/conditions:linux_x86_64": ["@remote_java_tools_linux//" + target],
+        "//src/conditions:linux_aarch64": ["@remote_java_tools_linux//" + target],
+        "//src/conditions:darwin": ["@remote_java_tools_darwin//" + target],
```

Comment:
What's the difference between `darwin` and `darwin_x86_64`?

Reply:
Wise Marcel explained that `darwin` is used on Mac when there is no xcode installed and `darwin_x86_64` is used on a mac when there is xcode. 

A prebuilt binary on `darwin` works on `darwin_x86_64` but not the other way around.

The binary was `darwin_x86_64`. I recompiled the archive on `darwin` and updated the references.